<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>endingWindow</class>
 <widget class="QWidget" name="endingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(
    x1: 0, y1: 0, x2: 1, y2: 1,
    stop: 0 rgba(196, 236, 236, 1), stop: 1 rgba(255, 255, 255, 1)
);</string>
  </property>
  <widget class="QFrame" name="endingWindow_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(
    x1: 0, y1: 0, x2: 1, y2: 1,
    stop: 0 rgba(196, 236, 236, 1), stop: 1 rgba(255, 255, 255, 1)
);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="readyLable">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;
background:none;
border: none;
color: black;</string>
    </property>
    <property name="text">
     <string>Your music sheet is ready!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="feedback">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
border: none;
color: black;
font: 13pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>We appreciate your feedback:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
border: none;
color: black;</string>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="emailButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Arial&quot;;
background:none;
border: none;
color: black;
</string>
    </property>
    <property name="text">
     <string>Send to email</string>
    </property>
   </widget>
   <widget class="QFrame" name="starFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>600</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
border: none;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="star5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/starBlack.png</normaloff>../images/starBlack.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="star2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/starBlack.png</normaloff>../images/starBlack.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="star4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/starBlack.png</normaloff>../images/starBlack.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="star1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/starBlack.png</normaloff>../images/starBlack.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="star3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/starBlack.png</normaloff>../images/starBlack.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="thanksLabel">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>120</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
border: none;
color: black;
font: 24pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Thank you for using
 PerfectPitch!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="logoButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
border: none;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/icon.png</normaloff>../images/icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="backButtonEndingWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
border: none;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/back_light.png</normaloff>../images/back_light.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
